code/datums/diseases/advance/symptoms/fever.dm

```diff
@@ -36,6 +36,6 @@ Bonus
 	return
 
 /datum/symptom/fever/proc/Heat(mob/living/M, datum/disease/advance/A)
-	var/get_heat = (sqrt(21+A.totalTransmittable()*2))+(sqrt(20+A.totalStageSpeed()*3))
+	var/get_heat = (sqrt(max(0,21+A.totalTransmittable()*2)))+(sqrt(max(0,20+A.totalStageSpeed()*3)))

```

Comment:
Hmm, let me check i assumed there's simply some disease that sets these to some negative define